<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="byte_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="byte_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Byte</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kb_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="kb_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>KB</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mb_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mb_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MB</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gb_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="gb_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GB</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
